tycho-its/projects/target.artifact.junit/bundle-example/src/test/java/com/tycho/repro/MyExampleTestIT.java

```diff
@@ -0,0 +1,10 @@
+package com.tycho.repro;
+
+import org.junit.Test;
```

Comment:
Why using Junit4 here?

Reply:
@laeubi it's on purpose. On my project I have existing JUnit 4 tests that I don't want to touch